<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <include file="$(find rr_platform)/launch/bigoli_platform.launch"/>

    <node name="navigation_controller" pkg="rr_common" type="navigation_controller" output="screen">
        <param name="req_finish_line_crosses" type="int" value="2"/>
        <param name="startSignal" type="string" value="/start_detected"/>
        <param name="resetSignal" type="string" value="/reset_detected"/>
    </node>

    <include file="$(find rr_avc)/launch/mapper.launch"/>

    <include file="$(find rr_avc)/launch/planner.launch">
        <arg name="wheel_base" value="1.12"/>
        <arg name="lateral_accel" value="1.0"/>
        <arg name="max_speed" value="1.5"/>
        <arg name="collision_dist_front" value="1.1"/>
        <arg name="collision_dist_back" value="0.2"/>
        <arg name="collision_dist_side" value="0.4"/>
        <arg name="steer_limit" value="0.25"/>
        <arg name="input_cloud_topic" value="/current_obstacles"/>
        <arg name="smoothing_array_size" value="3"/>
        <arg name="left_dist_weight" value="1"/>
        <arg name="right_dist_weight" value="1"/>
        <arg name="obs_dist_slow_thresh" value="1000"/>
        <arg name="obs_dist_slow_ratio" value="1.0"/>
    </include>
    
    <include file="$(find rr_avc)/launch/start_detector_standalone.launch"/>
    <include file="$(find rr_avc)/launch/finish_detector.launch"/>
    <include file="$(find rr_common)/launch/sound_node.launch"/>
</launch>
